yogo/packages/core/src/app.e2e.test.ts

```diff
@@ -0,0 +1,125 @@
+import { bootstrap } from "./bootstrap";
+import { ApolloServer } from "apollo-server";
+import {
+  createTestClient,
+  ApolloServerTestClient
+} from "apollo-server-testing";
+
+describe("e2e", () => {
```

Comment:
This test actually boots up an HTTP graphql server on port:4000.
Since the whole picture of Dawlet and its API is still ambiguous, end-to-end testing is best suitable for the next several iterations of the development.